package.json

```diff
@@ -56,7 +56,7 @@
 		"@wp-playground/client": "^1.0.11",
 		"react": "^18.3.1",
 		"react-dom": "^18.3.1",
-		"react-router-dom": "^6.28.0",
```

Comment:
I'm not too confident with the upgrade to v7, looking at the docs, there's seems to be some changes that might be needed for the upgrade to work without issues: https://reactrouter.com/upgrading/v6

Reply:
I did test the functionality with v7 and nothing broke but if you want to stick to v6 before making the changes you suggested you wanted with the router, we can hold off from that in this PR.

Reply:
Reverting for later.